Data-Structures/src/main/java/Data/Structures/LinkedList/Llist.java

```diff
@@ -0,0 +1,49 @@
+package Data.Structures.LinkedList;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Llist {
+    public  Node head;
+    public  Node current;
+
+    public Llist(){
+
+    }
+
+   //Insert a value as a node
+   public void insert(int value){
+       Node insertNode = new Node(value);
+       insertNode.next= head;
+       head = insertNode;
+   }
+
+
+    //check if the value exists within the list
+    public boolean includes(int value){
+      current = head;
+      while(current.next!=null){
+          if(current.value == value){
+              return true;
+          }
+          current = current.next;
+      }
+        if(current.value == value) return true;
+        else return false;
+    }
+
+
+   //return the current node
+    public List<Node> print(){
+
```

Comment:
There's an empty line here, in the other methods there isn't an empty line after the function declaration. Either is fine, but you should strive for consistency (either give all of them or none of them an empty line)

Reply:
line 38?